js/markDone.js

```diff
@@ -0,0 +1,21 @@
+import { EDIT, COMPLETE } from "/js/constants.js";
+import calculateDuration from "/js/utility/mDDuration.js";
+import createDurationElement from "/js/utility/mDDuElement.js";
+
+export default function markTodoAsCompleted(element, taskList, id) {
+  const index = taskList.findIndex((item) => item.id === id);
+  taskList[index].done = true;
+
+  element.querySelector(".text").style.textDecoration = "line-through";
+  element.querySelector(".text").style.color = "green";
+  element.querySelector(`[data-job="${COMPLETE}"]`).style.display = "none";
+  element.querySelector(`[data-job="${EDIT}"]`).style.display = "none";
+
+  const taskStartday = element.querySelector(".date").innerText;
+  let start = taskStartday.replace(/[^0-9]/g, "-");
+  start = start.slice(12);
```

Comment:
What does this `start` mean? Does it represent the start date of a task in a string? If Yes we can write this as:
`let taskStartDateString = taskStartday.replace(/[^0-9]/g, "-").slice(12);`

Reply:
yes, I fetch the start date of a task in a string form first then parse it to date. I have renamed it as you suggested.